dev-util/qdevicemonitor/qdevicemonitor-9999.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="6"
+
+inherit qmake-utils
+
+DESCRIPTION="Crossplatform log viewer for Android, iOS and text files"
+HOMEPAGE="https://github.com/alopatindev/qdevicemonitor"
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/alopatindev/qdevicemonitor"
+else
+	SRC_URI="https://github.com/alopatindev/qdevicemonitor/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
you need to put `KEYWORDS` in the else clause above, `-9999` ebuilds cant be KEYWORDed